features/step_definitions/file_steps.coffee

```diff
@@ -4,18 +4,22 @@ path = require 'path'
 
 module.exports = ->
 
-  @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+  @Given /^my project has no test files$/, (done) ->
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^my project has a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
Can these just go back to single lines? They fit and it brought a disagreement that I doubt either of us will back down on.